main.go

```diff
@@ -29,6 +29,57 @@ func runCli() error {
 					},
 				},
 			},
+			{
+				Name:   "tweaks",
+				Usage:  "manage Ultramarine tweaks, a set of optional system patches and configurations",
+				Action: listTweaks,
+				Subcommands: []*cli.Command{
+					{
+						Name:    "enable",
+						Usage:   "Enable an Ultramarine tweak",
```

Comment:
```suggestion
						Usage:   "Enable an Ultramarine Tweak",
```
are we branding tweaks as a feature?? if yes it should be stylized uppercase

Reply:
Yeah, it's a feature, capitalize Tweaks everywhere.